zeebe/gateway-protocol/src/main/proto/rest-api.yaml

```
@@ -9029,6 +9029,10 @@ components:
           description: Total items matching the criteria.
           type: integer
           format: int64
+        hasMoreTotalItems:
+          description: |
+            Indicates if more results exist beyond the reported totalItems value. If ElasticSearch or OpenSearch is used, the totalItems value is capped at 10,000 due to search limitations. This field is true if additional results exist beyond this cap; otherwise, it is false.
```

💭 Same as mentioned above, we should keep this more generic here.
```suggestion
            Indicates if more results exist beyond the reported totalItems value. Due to system limitations, the totalItems value can be capped.
```